packages/legend-application-studio/src/components/editor/editor-group/service-editor/testable/ServiceTestDataEditor.tsx

```diff
@@ -201,6 +455,15 @@ export const ConnectionTestDataEditor = observer(
       }
     };
 
+    const generateTestDataWithSeedData = (): void => {
+      connectionTestDataState.setUseSeedDataInputModal(true);
+      connectionTestDataState.setNewTableIdentifierState([]);
+      const tables = connectionTestDataState.getAvailableTables();
+      if (tables[0]) {
+        connectionTestDataState.addNewTableIdentifierState(tables[0]);
+      }
```

Comment:
```suggestion
      const tables = connectionTestDataState.getAvailableTables();
      if (tables[0]) {
        connectionTestDataState.addNewTableIdentifierState(tables[0]);
      }
```
      const table = getNullableFirstEntry(connectionTestDataState.getAvailableTables());
      if (table) {
        connectionTestDataState.addNewTableIdentifierState(table);
      }